staging/src/kubevirt.io/api/core/v1/types.go

```diff
@@ -2836,6 +2836,11 @@ type InterfaceBindingPlugin struct {
 	// version: v1alphav1
 	// +optional
 	DownwardAPI NetworkBindingDownwardAPIType `json:"downwardAPI,omitempty"`
+
+	// ComputeResourceOverhead specifies the resource overhead that should be added to the compute container when using the binding.
+	// version: v1alphav1
+	// +optional
+	ComputeResourceOverhead *k8sv1.ResourceRequirements `json:"computeResourceOverhead,omitempty"`
```

Comment:
Hello @Sreeja1725, when declaring this member an object (and not as a pointer), the API backwards compatibility unit test fails.
Do you maybe know why?

Reply:
Thank you for your answer @Sreeja1725.